### DISCUSSION TITLE
Import from spreadsheet or csv

### DISCUSSION THREAD

**Discussion by kosielemmer** [Donkie/Spoolman#402] [Ideas]
Hi

Just installed and the functionality is great.  What I was thinking was whether it would be possible to create a spreadsheet template that can then be imported as a data take on activity just after installing for the first time.

**Comment by jeffalanmcclain** [comment]
You can pull the SQLite DB file over into any SQLite viewer / editor file and enter all the relevant things in there (or any SQLite CSV import function).

**Reply by daFreeMan** [reply]
What would be the username/pwd to gain access to the SQLite DB file?
